static/docs/get-started/import-data.md

```diff
@@ -0,0 +1,72 @@
+# Import Data
+
+We've seen how to [push](/doc/get-started/store-data) and
+[pull](/doc/get-started/retrieve-data) data a <abbr>DVC project</abbr>'s
+[remote](/doc/commands-reference/remote). But what if we wanted to integrate a
+dataset or ML model produced in one project into another project (as an
+[external dependency](/doc/user-guide/external-dependencies))?
+
+One way is to download the data (with `wget` or `dvc get`, for example) and use
+`dvc add` to track it, but the connection between projects would be lost. We
+wouldn't be able to tell where the data came from or whether there are new
+versions available.
+
+There's a better alternative (for projects in a Git repository): the
```

Comment:
a few problems here - `for projects` - does it mean that import does not work for other projects? or that the project we are importing from should be a git project. 

I would imagine 99% of the DVC projects are Git projects - that's why I'm not sure we need to mention this note here. May be it's better to mention something like `... by providing a Git URL to a DVC project`?

Reply:
Good point. How about "It downloads data from any DVC project in a Git repository...'? (Again, this wording is copied from the `dvc import` cmd ref.) Changed in fb4efc8.